presto-expressions/src/main/java/com/facebook/presto/expressions/DynamicFilters.java

```diff
@@ -60,6 +60,26 @@ public static DynamicFilterExtractResult extractDynamicFilters(RowExpression exp
         return new DynamicFilterExtractResult(staticConjuncts.build(), dynamicConjuncts.build());
     }
 
+    public static DynamicFilterExtractRowExpression extractDynamicFiltersRowExpression(RowExpression expression)
```

Comment:
Let's just do two interfaces:
- `RowExpression extractDynamicConjuncts(RowExpression expression)` and
- `RowExpression extractStaticConjuncts(RowExpression expression)`